GliaWidgets/Sources/Extensions/Data+Extension.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Data {
+    var html2AttributedString: NSAttributedString? {
+        do {
+            return try NSAttributedString(
+                data: self,
+                options: [.documentType: NSAttributedString.DocumentType.html, .characterEncoding: String.Encoding.utf8.rawValue],
```

Comment:
NIT: This line could be made shorter by either adding a new line on each param, or if it becomes too nested, then the options could be generated as a variable before the return and then just use the variable

```
var options: [whatever type] = [
    .documentType:...,
    ...
]

return try NSAttributedString(
    data: self,
    options: options,
    ....
)
````

Reply:
4b3ba3b61831c87a2f3124f2c5ea240f45e7d75a